docs/admin/kubeadm-alpha.md

```diff
@@ -0,0 +1,266 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm alpha
+notitle: true
+---
+
+# Kubeadm alpha
+`kubeadm alpha` provides a preview of a set of features made available for gathering feedback
+ from the community. Please try it out and give us feedback!
+{: .caution}
+
+In v1.8.0, kubeadm introduced `kubeadm alpha phase` command with the aim to make kubeadm more 
+modular, letting you invoke atomic sub-steps of the bootstrap process; basically, you can 
+let kubeadm do some parts and fill in yourself where you need customizations.
+
+`kubeadm alpha phase` is consistent with [kubeadm init workflow](/docs/admin/kubeadm-init.md#init-workflow), 
+and behind the scene both use the same code.
+
+* [Kubeadm alpha phase preflight](#cmd-phase-preflight)
+* [Kubeadm token create](#cmd-phase-token)
+* [Kubeadm alpha phase certs](#cmd-phase-certs)
+* [Kubeadm alpha phase kubeconfig](#cmd-phase-kubeconfig)
+* [Kubeadm alpha phase controlplane](#cmd-phase-controlplane)
+* [Kubeadm alpha phase etcd](#cmd-phase-etcd)
+* [Kubeadm alpha phase mark-master](#cmd-phase-mark-master)
+* [Kubeadm alpha phase bootstrap-token](#cmd-phase-bootstrap-token)
+* [Kubeadm alpha phase upload-config](#cmd-phase-upload-config)
+* [Kubeadm alpha phase addon](#cmd-phase-addon)
+* [Kubeadm alpha phase selfhosting](#cmd-phase-self-hosting)
+
+## Kubeadm alpha phase preflight {#cmd-phase-preflight}
+
+You can execute preflight checks both for master node, like in `kubeadm init`, or for worker node
+like in `kubeadm join`.
+
+{% capture preflight_master %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_preflight_master.md %}
+{% endcapture %}
+
+{% capture preflight_node %}
+{% include_relative _kubeadm/kubeadm_alpha_phase_preflight_node.md %}
+{% endcapture %}
+
+{% assign tab_set_name = "tab-preflight" %}
+{% assign tab_names = "master,node" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: preflight_master | push: preflight_node %}
+
+{% include tabs.md %}
+
+## Kubeadm token create {#cmd-phase-token}
+
+Currently there is no a specific command under `kubeadm alpha phase` for creating a token; however
```

Comment:
Change "Currently" to something like "In kubeadm alpha".

use kubeadm token create command -> use the kubeadm token create command